soccer/gameplay/skills/dribble.py

```diff
@@ -97,18 +97,21 @@ def on_exit_capture(self):
         self.remove_subbehavior('capture')
 
     def execute_aim(self):
-        self.robot.set_max_angle_speed(3)
+        self.robot.set_max_angle_speed(2)
         self.robot.pivot(self._pos)
         self.robot.set_dribble_speed(self._dribble_speed)
         if self.robot.has_ball():
             self.last_ball_time = time.time()
 
     def execute_drive(self):
         self.robot.set_dribble_speed(self._dribble_speed)
-        self.robot.face(self._pos)
+        self.robot.face(self.pos)
+
+        #self.robot.set_max_speed(.5)
 
         #offset by the size of the robot so the ball is on the target position when it stops
-        self.robot.move_to_direct(self._pos - (
+        self.robot.disable_avoid_ball()
+        self.robot.move_to(self._pos - (
```

Comment:
You use both `self._pos` and `self.pos` in this method - generally with properties, the setter is the only place that references the instance variable directly (`_pos`), while everywhere else uses the property accessor (`pos`).